metacat-main/src/main/java/com/netflix/metacat/main/configs/SNSNotificationsConfig.java

```diff
@@ -52,18 +45,14 @@
     /**
      * If SNS notifications are desired and no existing client has been created elsewhere
      * in the application create a default client here.
-     * @param config       The system configuration abstraction to use
-     * @param registry     registry for spectator
+     *
      * @return The configured SNS client
      */
     //TODO: See what spring-cloud-aws would provide automatically...
     @Bean
-    @ConditionalOnMissingBean(AmazonSNSAsync.class)
-    public AmazonSNSAsync amazonSNS(final Config config, final Registry registry) {
-        final ExecutorService executor = Executors.newFixedThreadPool(config.getSnsClientThreadCount(),
-            new ThreadFactoryBuilder().setNameFormat("metacat-sns-pool-%d").build());
-        RegistryUtil.registerThreadPool(registry, "metacat-sns-pool", (ThreadPoolExecutor) executor);
-        return new AmazonSNSAsyncClient(DefaultAWSCredentialsProviderChain.getInstance(), executor);
+    @ConditionalOnMissingBean
```

Comment:
I think you lost the class qualifier here. It helps cause otherwise I think it defaults to looking for a bean named `amazonSNS` and doesn't necessarily care about the class type.